src/main/java/jewellery/inventory/dto/request/ResourceInUserRequestDto.java

```diff
@@ -12,14 +12,14 @@
 @AllArgsConstructor
 @SuperBuilder
 public class ResourceInUserRequestDto {
-  private static final String QUANTITY_MIN_VALUE_MSG = "Quantity should not be less than 1.";
+  private static final String QUANTITY_MIN_VALUE_MSG = "Quantity should not be less than 0.01.";
   private static final String QUANTITY_DECIMAL_PLACES_MSG =
-      "Quantity should not have more than 2 decimal places.";
+          "Quantity should not have more than 2 decimal places.";
 
   @NotNull private UUID userId;
   @NotNull private UUID resourceId;
 
-  @Min(value = 0, message = QUANTITY_MIN_VALUE_MSG)
+  @DecimalMin(value = "0.01", message = QUANTITY_MIN_VALUE_MSG)
```

Comment:
it might be good to declare it in a constant as in "ResourceQuantityRequestDto"

Reply:
Done!